src/translations/nl_NL.translation

```diff
@@ -1,1361 +1,1361 @@
     </customLabels>
     <customLabels>
-        <label>Kopieertaak cursusbeschrijving voltooid met status:</label>
         <name>stgCourseDescriptionCopyEmailSubject</name>
+        <label>Kopieertaak cursusbeschrijving voltooid met status:</label>
     </customLabels>
     <customLabels>
-        <label>Standaardbeheersing taal contactpersoon</label>
         <name>stgDefaultContactLanguageFluency</name>
+        <label>Standaardbeheersing taal contactpersoon</label>
     </customLabels>
     <customLabels>
-        <label>Standaard recordtype voor staflid</label>
         <name>stgDefaultFacultyTypeTitle</name>
+        <label>Standaard recordtype voor staflid</label>
     </customLabels>
     <customLabels>
-        <label>Standaard recordtype voor actieve student</label>
         <name>stgDefaultStudentTypeTitle</name>
+        <label>Standaard recordtype voor actieve student</label>
     </customLabels>
     <customLabels>
-        <label>Foutafhandeling uitschakelen</label>
         <name>stgDisableErrorHandlingTitle</name>
+        <label>Foutafhandeling uitschakelen</label>
     </customLabels>
     <customLabels>
-        <label>Afdwinging voorkeurs-e-mail uitschakelen</label>
         <name>stgDisablePreferredEmailEnforcement</name>
+        <label>Afdwinging voorkeurs-e-mail uitschakelen</label>
     </customLabels>
     <customLabels>
-        <label>Afdwinging voorkeurstel.nr uitschakelen</label>
         <name>stgDisablePreferredPhoneEnforcement</name>
+        <label>Afdwinging voorkeurstel.nr uitschakelen</label>
     </customLabels>
     <customLabels>
-        <label><!-- EDA Setup title --></label>
         <name>stgEDASetupTitle</name>
+        <label>EDA-set-up</label>
     </customLabels>
     <customLabels>
-        <label>Cursusverbindingen inschakelen</label>
         <name>stgEnableCourseConnectionsTitle</name>
+        <label>Cursusverbindingen inschakelen</label>
     </customLabels>
     <customLabels>
-        <label>Verbeterde functionaliteit voorkeurstelefoonnummer inschakelen</label>
         <name>stgEnablePreferredPhoneSync</name>
+        <label>Verbeterde functionaliteit voorkeurstelefoonnummer inschakelen</label>
     </customLabels>
     <customLabels>
-        <label>Foutmeldingen aan:</label>
         <name>stgErrorNotiTo</name>
+        <label>Foutmeldingen aan:</label>
     </customLabels>
     <customLabels>
-        <label>Ontvangers foutmeldingen</label>
         <name>stgErrorNotifRecipientsTitle</name>
+        <label>Ontvangers foutmeldingen</label>
     </customLabels>
     <customLabels>
-        <label>Backfill van etniciteit en ras voor contactpersonen</label>
         <name>stgEthnicityRaceBackfillContacts</name>
+        <label>Backfill van etniciteit en ras voor contactpersonen</label>
     </customLabels>
     <customLabels>
-        <label>Vloeiend</label>
         <name>stgFluent</name>
+        <label>Vloeiend</label>
```

Comment:
This field should not be translated. It is a system field used only in the install script and refers to an API name.
```suggestion
        <label>Fluent</label>
```